bin/install_cmdstan

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+"""
+Download and install a CmdStan release from GitHub.
+Optional command line arguments:
+   -v, --version : version, defaults to latest
+   -d, --dir : install directory, defaults to '~/.cmdstanpy
+"""
+import argparse
+import contextlib
+import os
+import os.path
```

Comment:
This is not needed, `os` imports `os.path` by default

Reply:
thanks - will clean up!